<launch>
    <node name="ros_four1_talker_node" pkg="ros_four1_msg" type="ros_four1_talker" output="screen"/>
</launch>
